splinterd/src/daemon.rs

```diff
@@ -358,6 +358,7 @@ impl SplinterDaemon {
                 routes::get_status(node_id.clone(), service_endpoint.clone())
             }))
             .add_resource(routes::make_fetch_node_resource(node_registry.clone()))
+            .add_resource(routes::make_update_node_resource(node_registry.clone()))
```

Comment:
Typo in commit message "PATH /nodes/{identity}" -> "PATCH  /nodes/{identity}"

Reply:
Fixed